doc/mimalloc-doc.h

```diff
@@ -730,6 +730,32 @@ bool mi_heap_check_owned(mi_heap_t* heap, const void* p);
 /// @see mi_heap_contains_block()
 /// @see mi_heap_get_default()
 bool mi_check_owned(const void* p);
+/// Type of deferred free functions.
```

Comment:
You should add a blank line before `Type of deferred free functions`, otherwise Doxygen might generate incorrect documentation.

Reply:
Hmm I added blank line there but this review isn't shown as outdated?